tests/e2e/tests/content-manager/editview.spec.ts

```diff
@@ -15,51 +15,50 @@ test.describe('Edit View', () => {
       /\/admin\/content-manager\/collection-types\/api::article.article\/create(\?.*)?/;
     const LIST_URL = /\/admin\/content-manager\/collection-types\/api::article.article(\?.*)?/;
 
-    test.fixme(
-      'as a user I want to be warned if I try to publish content that has draft relations',
-      async ({ page }) => {
-        await page.getByLabel('Content Manager').click();
-        await page.getByRole('link', { name: 'Create new entry' }).click();
+    test('as a user I want to be warned if I try to publish content that has draft relations', async ({
+      page,
+    }) => {
+      await page.getByLabel('Content Manager').click();
+      await page.getByRole('link', { name: 'Create new entry' }).click();
 
-        // Wait for the URL to match the CREATE_URL pattern
-        await page.waitForURL(CREATE_URL);
+      // Wait for the URL to match the CREATE_URL pattern
+      await page.waitForURL(CREATE_URL);
 
-        // Add a new relation to the entry
+      // Add a new relation to the entry
 
-        await page.getByRole('combobox', { name: 'authors' }).click();
-        await page.getByLabel('Coach BeardDraft').click();
-        // Attempt to publish the entry
-        await page.getByRole('button', { name: 'Publish' }).click();
+      await page.getByRole('combobox', { name: 'authors' }).click();
+      await page.getByLabel('Coach BeardDraft').click();
+      // Attempt to publish the entry
+      await page.getByRole('button', { name: 'Publish' }).click();
 
-        // Verify that a warning about a single draft relation is displayed
-        await expect(page.getByText('This entry is related to 1')).toBeVisible();
-        await page.getByRole('button', { name: 'Cancel' }).click();
+      // Verify that a warning about a single draft relation is displayed
+      await expect(page.getByText('This entry is related to 1')).toBeVisible();
+      await page.getByRole('button', { name: 'Cancel' }).click();
```

Comment:
I thought we removed the count to draft relations, as we had a couple of issues with that

Reply:
See other comment. I turned them on but maybe I should be commenting this check here instead. Lmk what you think.